source/server-selection/server-selection.rst

```diff
@@ -1789,10 +1747,13 @@ References
 .. _Driver Authentication: https://github.com/mongodb/specifications/blob/master/source/auth
 .. _maxConnecting: /source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#connection-pool
 .. _Connection Monitoring and Pooling: /source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst
+.. _Global Command Argument: /source/message/OP_MSG.rst#global-command-arguments
 
 Changes
 =======
 
+2021-08-05: Updated $readPreference logic to describe OP_MSG behavior.
```

Comment:
If we do scrub this spec of SecondaryOk and OP_QUERY mentions, I think that should also be captured in this changelog entry.